static/src/javascripts/projects/common/modules/commercial/creatives/template-preprocessor.js

```diff
@@ -43,6 +54,41 @@ define([
     var manualContainerCtaSoulmatesTpl;
     var manualContainerCtaMembershipTpl;
 
+    function preprocessLogo(tpl) {
```

Comment:
Yeah I did this originally to avoid the `template` module to grow too large... I also hope we can get rid of much of what is in this module later (by doing a better job with parameters on the DFP side).